docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+Objects, on the other hand, are specific to locations in a frame.
+For example, if you want to annotate where is the dog in the image, you would first add an object to the ontology.
+Objects could for example be a bounding box or a polygon indicating where the dog is.
+In the following subsections, we will show you how both scenarios are done.
+
+
+Adding a classification to a project ontology
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Here, we are in the "cat vs. dog" scenario from above.
+Adding classifications for cats and dogs to an ontology via the |sdk| is done as follows.
+
+The :meth:`add_classification() <.EncordClientProject.add_classification>` method takes the following parameters:
+
+* ``name`` the description of the classification
+* ``classification_type`` a value from the :class:`.ClassificationType` enum.
+* ``required`` - a boolean value specifying whether the classification is mandatory
+* ``options`` - a list of options that the annotator can choose from in case of :class:`CHECKLIST <.ClassificationType>`/:class:`RADIO <.ClassificationType>`.
+  Omit this parameter if adding a ``TEXT`` classification.
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/projects/adding_classifications_to_a_project_ontology.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            True  # False if unsuccessful
+
+
+
+Adding an object to a project ontology
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Here, we are in the "where is the dog" scenario from above.
+Adding objects to an existing ontology via the |sdk| is demonstrated in the following example.
+The :meth:`add_object() <.EncordClientProject.add_object>` method takes the following parameters:
+
+* ``name`` the name of the object as a string,
+* ``shape`` the shape of the object, which is an :class:`.ObjectShape` enum.
+
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/projects/adding_objects_to_a_project_ontology.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            True  # False if unsuccessful
+
+
+Labels
+======
+
+.. note::
+    This section is undergoing a reconstruction [Tuesday, May 10th 2022].
+
+A label row has the attributes ``<label_hash>`` uid, ``data_title``, ``data_type``, ``data_units``, and ``label_status``.
+
+.. code-block::
+
+    {
+        'label_hash': label_hash (uid),
+        'data_title': data_title,
+        'data_type': data_type,
+        'data_units': data_units,
+        'object_answers': object_answers,
+        'classification_answers': classification_answers,
+        'label_status': label_status,
+    }
+
+A label row contains a data unit or a collection of data units and associated labels, and is specific to a data asset with type video or image group.
+
+1.  A label row with a data asset of type ``video`` contains a single data unit.
```

Comment:
```suggestion
1.  A label row with a data asset of type ``video`` contains a single data point
```

Reply:
the term data unit is used all over the place in keys for accessing the data so I think that it would be more clear to stick with that?